### ISSUE TITLE
Not yet fully anon

### ISSUE THREAD

**Issue by kappapiana** [kappapiana/anonymize#18]
After running the script, still creator information remains there.

For docx:

`docProps/core.xml:<cp:coreProperties xmlns:cp="http://schemas.openxmlformats.org/package/2006/metadata/core-properties" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:dcmitype="http://purl.org/dc/dcmitype/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><dcterms:created xsi:type="dcterms:W3CDTF">2022-02-16T16:03:46Z</dcterms:created><dc:creator>XXXX</dc:creator><dc:description></dc:description><dc:language>en-GB</dc:language><cp:lastModifiedBy>YYYY</cp:lastModifiedBy><dcterms:modified xsi:type="dcterms:W3CDTF">2022-02-16T16:06:38Z</dcterms:modified><cp:revision>2</cp:revision><dc:subject></dc:subject><dc:title></dc:title></cp:coreProperties>`

## How to fix it

I think we should add a further deletion as with "delete initials" and "delete dates", without replacing the information therein

**Comment by yamanq** [comment]
Yes, and the delete_dates function should also target the date created and date modified tags.

I found [this](https://www.ecma-international.org/publications-and-standards/standards/ecma-376/) resource that prints out all the properties in the standard (Part 2, pg 41), but we should also check various formats manually.
